apps/masterbots.ai/app/api/wordware/describe/route.ts

```
@@ -0,0 +1,58 @@
+      `https://app.wordware.ai/api/prompt/${promptId}/describe`,
+      {
+        headers: {
+          Authorization: `Bearer ${apiKey}`
+        }
+      }
+    )
+
+    const contentType = response.headers.get('content-type')
+    if (contentType && contentType.indexOf('application/json') !== -1) {
+      const data = await response.json()
+      const data = await response.json()
```

Biome

[error] Shouldn't redeclare 'data'. Consider to delete it or rename it.

'data' is defined here:


(lint/suspicious/noRedeclare)